README.md

```diff
@@ -94,3 +94,15 @@ Test specific files
 ```bash
 yarn test /test/**/constructor.spec.ts
 ```
+
+## 🙌🏾 Troubleshooting
+
+### Symlinking in Monorepo w/ Metro (`@magic-sdk/react-native` Package)
+
+For React Native projects living within a monorepo that run into the following `TypeError: Undefined is not an object` error: 
```

Comment:
Could you make **monorepo** bold?

And let's make a link to the RN readme instead of putting rn contents here.